daxgen/daxgen.py

```diff
@@ -42,14 +43,16 @@ def read(self, filename):
         methods = {
             'json': read_json,
             'gexf': nx.read_gexf,
+            'gxf': nx.read_gexf,
+            'gml': nx.read_graphml,
             'graphml': nx.read_graphml
         }
         ext = filename.split('.')[-1]
         try:
             self.graph = methods[ext.lower()](filename)
```

Comment:
at first I thought the Exception raised by the reader would be hidden behind this try, and the message would be lost but it would raise anyways as this only captures the KeyError. I think this is fine,